activemodel/lib/active_model/validations.rb

```diff
@@ -45,6 +45,26 @@ module Validations
       extend  HelperMethods
       include HelperMethods
 
+      ##
+      # :method: validation_context
+      # Returns the context when running validations.
+      #
+      # This is useful when running validations except a certain context (opposite to the +on+ option).
+      #
+      #   class Person
+      #     include ActiveModel::Validations
+      #
+      #     attr_accessor :name
+      #
```

Comment:
✂️ please remove this line to make the `Person` class style consistent with other examples.

Reply:
@yahonda removed it!